start.sh

```diff
@@ -35,15 +23,13 @@ cd /mapproxy
 mapproxy-util create -t wsgi-app -f mapproxy.yaml /mapproxy/app.py
 RELOAD_LOCKFILE=/settings/.app.lock
 if [[ ! -f ${RELOAD_LOCKFILE} ]];then
-  sed -i 's/'\)/', reloader=True\)/g' app.py
+  sed -i 's/\(, reloader=True\)*'\)'/, reloader=True\)/g' app.py
```

Comment:
Bug from the previous code:

In docker-compose where the container is not stateless, repeatedly running the same container will make the uwsgi fails because sed appended something like `, reloader=True, reloader=True)` due to the incomplete regex.

In k8s this is non-issue because pod are either run or deleted and only restarts if the first attempt fails.

The new regex explaination: if we have string `)` or multiple `, reloader=True`, then replace it as `, reloader=True)` once only.